packages/framework-info/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# CONTRIBUTING
+
+Contributions are always welcome, no matter how large or small. Before contributing, please read the
+[code of conduct](CODE_OF_CONDUCT.md).
+
+## Setup
+
+From the root of the repository:
+
+```sh
+npm install
+npm run build -- --scope=@netlify/framework-info
+npm test -- --scope=@netlify/framework-info
+```
+
+You can also use yarn.
+
+## Releasing
+
+Merge the release PR
+
+### Creating a prerelease
+
+1. Create a branch named `releases/<tag>/<version>` with the version and tag you'd like to release.
```

Comment:
it should be `releases/framework-info/rc.0/4.0.0` otherwise the release does not now which package to release